source/retryable-reads/tests/README.rst

```diff
@@ -0,0 +1,158 @@
+=====================
+Retryable Reads Tests
+=====================
+
+.. contents::
+
+----
+
+Introduction
+============
+
+The YAML and JSON files in this directory tree are platform-independent tests
+that drivers can use to prove their conformance to the Retryable Reads spec.
+
+Prose tests, which are not easily expressed in YAML, are also presented
+in this file. Those tests will need to be manually implemented by each driver.
+
+Tests will require a MongoClient created with options defined in the tests.
+Integration tests will require a running MongoDB cluster with server versions
+4.2.0 or later. N.B. Strictly speaking, the server should implement the
+"Early Failure on Socket Disconnect" slated for 4.2.0, but this is not
+necessary for initial testing.
+
+
+Server Fail Point
+=================
+
+See: `Server Fail Point <../../transactions/tests#server-fail-point>`_ in the
+Transactions spec test suite.
+
+Disabling Fail Point after Test Execution
+-----------------------------------------
+
+After each test that configures a fail point, drivers should disable the
+``failCommand`` fail point to avoid spurious failures in
+subsequent tests. The fail point may be disabled like so::
+
+    db.runCommand({
+        configureFailPoint: "failCommand",
+        mode: "off"
+    });
+
+Network Error Tests
+===================
+
+Network error tests are expressed in YAML and should be run against a standalone,
+shard cluster, or single-node replica set. Until `SERVER-34943`_ is resolved,
+these tests cannot be run against a multi-node replicaset.
```

Comment:
The change streams tests cannot be run on a standalone. I think we can add a tag to only run those tests on sharded clusters and replica sets. Maybe we can adopt the `topology: ["replicaset", "sharded"]` format from [the change stream tests themselves](https://github.com/mongodb/specifications/tree/master/source/change-streams/tests#spec-test-format)?

Reply:
I like that: I've taken your suggestion!